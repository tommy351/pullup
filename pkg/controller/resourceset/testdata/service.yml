---
apiVersion: pullup.dev/v1alpha1
kind: ResourceSet
metadata:
  name: test-46
  namespace: test
  uid: c37ee64f-5c69-4c53-8585-adc137d0acca
spec:
  number: 46
  resources:
    - apiVersion: v1
      kind: Service
      metadata:
        name: test
---
apiVersion: v1
kind: Service
metadata:
  name: test
  namespace: test
spec:
  clusterIP: 10.0.0.10
  ports:
    - nodePort: 32100
      port: 80
      protocol: TCP
      targetPort: 80
  selector:
    app: test
  type: NodePort
