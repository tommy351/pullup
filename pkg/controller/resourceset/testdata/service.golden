# This is a golden file. DO NOT EDIT.
/v1, Kind=Service:
  test/test-46:
    apiVersion: v1
    kind: Service
    metadata:
      name: test-46
      namespace: test
      ownerReferences:
      - apiVersion: pullup.dev/v1alpha1
        blockOwnerDeletion: true
        controller: true
        kind: ResourceSet
        name: test-46
        uid: c37ee64f-5c69-4c53-8585-adc137d0acca
    spec:
      ports:
      - port: 80
        protocol: TCP
        targetPort: 80
      selector:
        app: test
      type: NodePort
